src/Components/Artwork/GridItem.tsx

```diff
@@ -28,11 +28,52 @@ interface Props extends RelayProps, React.HTMLProps<ArtworkGridItemContainer> {
   }
 }
 
-class ArtworkGridItemContainer extends React.Component<Props, null> {
+interface State {
+  width: number
+  height: number
+}
+
+const GEMINI_HOST = "d7hftxdivxxvm.cloudfront.net"
+const IMAGE_QUALITY = 80
+
+class ArtworkGridItemContainer extends React.Component<Props, State> {
   static defaultProps = {
     useRelay: true,
   }
 
+  image: HTMLImageElement = null
+
+  state = {
+    width: 0,
+    height: 0,
+  }
+
+  componentDidMount() {
+    const { width } = this.image
+    const height = this.image.width * this.props.artwork.image.aspect_ratio
+
+    this.setState({
+      width,
+      height,
```

Comment:
Multiply them both by `window.devicePixelRatio`.

Reply:
This is small, but it adds up: 

<img width="99" alt="screen shot 2018-07-30 at 4 47 34 pm" src="https://user-images.githubusercontent.com/236943/43429435-52b74d9e-9418-11e8-9730-bf8bb79c2f0f.png">

vs

<img width="166" alt="screen shot 2018-07-30 at 2 36 58 pm" src="https://user-images.githubusercontent.com/236943/43429446-5d9ce9d0-9418-11e8-8fbc-e91c0b7bd49f.png">

Given the overall small dimensions of the grid items, is it worth it to double the image size for a slight increase in resolution? At a larger size (for example, the center image on `/artwork/id`) it's a bit more noticable.